Dockerfile

```diff
@@ -55,5 +56,11 @@ COPY . .
 # possible to run everything as root as well.
 RUN adduser --disabled-password --gecos "OpenWPM"  openwpm
 
+# Add Tini
```

Comment:
We need to tini, because bash ( the default PID 1) doesn't forward signals coming from the host to our container.
As since we can't pass the `--init` flag in a kubernetes crawl (or at least I didn't find a solution) I thought this would be the way to add it.
But just using an ubuntu tini base image seems better